app/code/Magento/Elasticsearch/README.md

```diff
@@ -1,2 +1,30 @@
-Magento\Elasticsearch module allows to use Elastic search engine for product searching capabilities.
-The module implements Magento\Search library interfaces.
\ No newline at end of file
+#Magento_Elasticsearch module
+
+Magento_Elasticsearch module allows using the Elasticsearch engine for the product searching capabilities.
```

Comment:
Please add a note that although it provides logic used by other modules implementing newer versions of Elasticsearch, this module by itself only adds support for Elasticsearch v5